feat(OHI IBM MQ): Added branching doc structure (#16140)
newrelic/docs-website: src/install/ibm-mq/linux/install-tarball.mdx

---
componentType: default
headingText: Download using tarball
---

>     <Callout variant="tip">
>       We recommend using your distribution package manager to [install the infrastructure agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/install-infrastructure-linux-using-package-manager). If you choose to [manually install the agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux), you may also want to install the on-host integrations from the provided tarballs.
>     </Callout>
> 
>     The manual install process isn't automated. If you install it manually, you must place the different files in the correct folders, and ensure that the agent has all the permissions to execute the integrations.

Reviewer: ```suggestion
    <Callout variant="tip">
      We recommend using your distribution package manager to [install the infrastructure agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/install-infrastructure-linux-using-package-manager). If you choose to [manually install the agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux), you may also want to install the on-host integrations from the provided tarballs.
      
      The manual install process isn't automated. If you install it manually, you must place the different files in the correct folders, and ensure that the agent has all the permissions to execute the integrations.
    </Callout>
```

    To install an integration from a tarball:

    1. Download the packaged integration file from the [tarball repository](https://download.newrelic.com/infrastructure_agent/binaries/linux/amd64/).

>     2. Unpack the tarball file according to our [integration file structure and placement rules](/docs/integrations/integrations-sdk/getting-started/integration-file-structure-activation), so the agent is able to find the definitions, configurations, and executables of the integration.

Reviewer: ```suggestion
    2. Unpack the tarball file according to our [integration file structure and placement rules](/docs/integrations/integrations-sdk/getting-started/integration-file-structure-activation) so the agent can find the definitions, configurations, and executables of the integration.
```

    3. Place the binary that contains the definition file inside `newrelic-integrations` or `custom-integrations` in the [agent directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#agent-directory).
    4. Place the integration's config file under the [plugin directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#configure-plugin).
    5. Follow the JMX tool instructions below.

### Use of the New Relic JMX tool [#nrjmx]

> Download the JMX tool from [our repository](https://download.newrelic.com/infrastructure_agent/binaries/linux/noarch/) and unpack it.

Reviewer: Replace step 5 with the JMX tool steps, and remove the section. Example:

5. Download the JMX tool from [our repository](https://download.newrelic.com/infrastructure_agent/binaries/linux/noarch/) and unpack it. Some notes on compatibility:
    * For JMX integration version 2.3.3 or higher and Cassandra integration version 2.3.0 or higher, the `nrjmx` tool is included as a dependency. For this reason, if you're using a package manager,  you do not need to install the `nrjmx` tool manually.
    * If you have `nrjmx` already installed and install `nri-jmx`, our JMX tool keeps the already-installed version. If you don't have `nrjmx` already installed, it gets the latest `nrjmx` release.
    * By default, the `nrjmx` location is `/usr/bin/nrjmx/*`. To install in a different location, set the new path in the `NR_JMX_TOOL` environment variable.
